LayoutTests/fast/speechsynthesis/voices-non-mock.html

```diff
@@ -23,7 +23,7 @@
             foundDefaultVoice = true;
     }
 
-    shouldBeTrue("voiceCount > 20");
+    shouldBeTrue(`voiceCount > ${window.internals.expectedVoiceCount}`);
```

Comment:
How about:

```suggestion
    shouldBeTrue(`voiceCount >= ${window.internals.expectedVoiceCount}`);
```

Or even:

```suggestion
    shouldBeTrue(`voiceCount >= ${window.internals.minimumExpectedVoiceCount}`);
```

Reply:
Why not? =) Let me come back with an updated patch.